src/main/java/com/linkmoa/domain/directory/entity/Directory.java

```diff
@@ -0,0 +1,82 @@
+package com.linkmoa.domain.directory.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.linkmoa.common.entity.BaseEntity;
+import com.linkmoa.domain.site.entity.Site;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+@Entity
+@Getter
+@Table(name = "directories")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Directory extends BaseEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "directory_id")
+	private Long id;
+
+	@Column(name = "directory_name")
+	private String directoryName;
```

Comment:
이미 테이블이나 엔티티명에서 이것이 디렉토리임을 잘 드러내고 있다고 생각하고, `Directory.directoryName`과 같은 반복이 생길 수 있을 것 같아서 아래와 같이 수정해보면 좋을 것 같습니다 :thinking: (이하 동일)

```suggestion
	@Column(name = "name")
	private String name;
```